src/stories/icon/icon-favourite/IconFavourite.tsx

```diff
@@ -0,0 +1,24 @@
+export type IconFavouriteProps = {
+  fill?: boolean;
+};
+
+export const IconFavourite = ({ fill }: IconFavouriteProps) => {
+  // This svg is a copy from public/icons/basic/icon-heart.svg.
+  // It is made as inline svg to be able to change fill value from props
+  return (
+    <svg
+      className="icon-favourite"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill={fill ? "#000" : "none"}
```

Comment:
Css is a weak spot of mine, but isn't it possible to define these attributes in css/sass instead?